xclim/core/options.py

```diff
@@ -22,6 +23,7 @@
     CF_COMPLIANCE: "warn",
     CHECK_MISSING: "any",
     MISSING_OPTIONS: {},
+    OUTPUT_DATASET: False,  # TODO : Change to True in 0.20
```

Comment:
You think the default should be to return a dataset ?

Reply:
Because we chose that multi-output Indicator return Datasets, I thought that _always_ returning a dataset was more consistent. But, your point on backward-compatibility is more important, I will remove the warning and this comment.